index.html

```diff
@@ -42,6 +42,148 @@ <h2 class="headline-fmt supporting-text">I am a Software Developer</h2>
+              <li>Ruby</li>
+            </ul>
+            <a class="textblock-link" href="#" target="_blank">See Project</a>
+
+          </div>
+        </div>
+        <div class="other-content">
+          <div class="content-defaults content1">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content2">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content3">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content4">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content5">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
```

Comment:
Kindly align the items to the left to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.
 *Figma Design*👇
![align](https://user-images.githubusercontent.com/73854999/198828524-587311dc-24f1-4704-b4bf-f9db50c86e90.PNG)

*Your design* 👇
![alin](https://user-images.githubusercontent.com/73854999/198828606-fed0d33e-0889-4801-8f9a-ac01e784b022.PNG)